src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -135,9 +135,11 @@ scheduling::NodeID ClusterResourceScheduler::GetBestSchedulableNode(
     int64_t *total_violations,
     bool *is_infeasible) {
   // The zero cpu actor is a special case that must be handled the same way by all
-  // scheduling policies, except for HARD node affnity scheduling policy.
+  // scheduling policies, except for HARD node affinity scheduling policy,
```

Comment:
Hmm I feel like it may not work actually. We should probably discuss the approach offline.